$NetBSD$

  - The functions are called filledPieColor() and filledPieRGBA() in the
    SDL_gfx library.

--- src/SDL.xs.orig	2005-10-05 11:25:49.000000000 +0200
+++ src/SDL.xs
@@ -3911,7 +3911,7 @@ GFXFilledpieColor ( dst, x, y, rad, star
     Sint16 end;
     Uint32 color;
 CODE:
-     RETVAL = filledpieColor( dst, x, y, rad, start, end, color );
+     RETVAL = filledPieColor( dst, x, y, rad, start, end, color );
 OUTPUT:
      RETVAL
 
@@ -3928,7 +3928,7 @@ GFXFilledpieRGBA ( dst, x, y, rad, start
     Uint8 b;
     Uint8 a;
 CODE:
-     RETVAL = filledpieRGBA( dst, x, y, rad, start, end, r, g, b, a );
+     RETVAL = filledPieRGBA( dst, x, y, rad, start, end, r, g, b, a );
 OUTPUT:
      RETVAL
 
